src/server/pkg/deploy/assets/postgres.go

```diff
@@ -347,26 +329,3 @@ func PostgresService(local bool, opts *AssetOpts) *v1.Service {
 		},
 	}
 }
-
-// PostgresInitConfigMap generates a configmap which can be mounted into
-// the postgres container to initialize the database.
-func PostgresInitConfigMap(opts *AssetOpts) *v1.ConfigMap {
-	return &v1.ConfigMap{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "ConfigMap",
-			APIVersion: "v1",
-		},
-		ObjectMeta: objectMeta(postgresInitConfigMapName, labels(postgresName), nil, opts.Namespace),
-		Data: map[string]string{
-			"init-db.sh": `
-#!/bin/bash
-set -e
-
-psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
-    CREATE DATABASE dex;
-    GRANT ALL PRIVILEGES ON DATABASE dex TO postgres;
```

Comment:
This was a hacky solution to set up the dex schema - dex does it's own migrations.